invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesCarousel/index.js

```diff
@@ -5,31 +5,67 @@
  * Invenio is free software; you can redistribute it and/or modify it
  * under the terms of the MIT License; see LICENSE file for more details.
  */
-import React from "react";
-import ReactDOM from "react-dom";
-import CommunitiesCarousel from "./CommunitiesCarousel";
-import { OverridableContext, overrideStore } from "react-overridable";
+import $ from "jquery";
```

Comment:
annoying guy mode on :)
Can we get rid of all the JS and use CSS only? [Example](https://css-tricks.com/css-only-carousel/) first link searching on the internet...

Reply:
Agree on that if it's possible. 

An alternative, in case it's too difficult/tricky to get right, would be to optimize for how the page loading looks like with "Slow 3G" throttling. If we get the first carousel "slide" to display ok, and get the interactivity later, maybe it's good enough? That's how it currently works for the legacy Zenodo frontpage.

Reply:
Tip: to achieve that, we can try to avoid any dependency and simply use plain JS (zero imports, no jquery). In this way, we will avoid depending on any bundle, and the JS can be downloaded and executed independently of other bundles.

Reply:
Testing with (fast) 3g (slow 3g takes forever to show anything at all - just a white page) the template is loading equally fast as the temporary "recent uploads" placeholders, but without any functionality for a while, so I guess that's what you suggested already?